web-configs/nginx/nginx-no-ssl.conf

```diff
@@ -0,0 +1,37 @@
+# Server block definition for cobalt web.
+server {
+    # Listen for for requests to the domain on port 80.
+    listen 80;
+    listen [::]:80;
+    server_name co.wukko.sh;
+	
+    # Set loggging. By default, logging is off for privacy.
```

Comment:
this is not necessarily true, as the default `nginx.conf` usually contains an `access_log` directive and it will just fall back to that
to actually turn off logging you would have to set `access_log off;` and `error_log /dev/null;`